doc/mgmt/SONiC_OC_PortChannel_Interface_HLD.md

```diff
@@ -0,0 +1,439 @@
++         +--rw oc-lag:aggregation
++         |  +--rw oc-lag:config
+          |  |  +--rw oc-lag:lag-type?    aggregation-type
++         |  |  +--rw oc-lag:min-links?   uint16
++         |  +--ro oc-lag:state
+          |  |  +--ro oc-lag:lag-type?    aggregation-type
++         |  |  +--ro oc-lag:min-links?   uint16
+          |  |  +--ro oc-lag:lag-speed?   uint32
++         |  |  +--ro oc-lag:member*      oc-if:base-interface-ref
++         |  |  +--ro if-agmnt:fallback?  boolean  //Augmented
+
+```
+
+**Note:** openconfig-if-aggregate.yang data model is augmented to support LACP Fallback.
+
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+
+### Create a PortChannel
+`interface PortChannel <channel-number>`
+```
+sonic(config)# interface PortChannel 1
+```
+### Configure min-links
+`minimum-links <number>`
+<br>
+ Default:0
+```
+sonic(config)# interface PortChannel 1
+sonic(conf-if-po1)# minimum-links 1
+```
+### Reset to default value of min-links
+`no minimum-links`
+<br>
+ Default:0
+```
+sonic(conf-if-po1)# no minimum-links 
+```
+### Configure MTU
+`mtu <mtu-val>`
+```
+sonic(conf-if-po1)#  mtu 9000
+sonic(conf-if-po1)# no mtu
+```
+#### MTU Removal
+`no mtu` --> Reset to default
+```
+sonic(conf-if-po1)# no mtu
+```
+
+#### Enable PortChannel
+`no shutdown` 
+```
+sonic(conf-if-po1)# no shutdown
+```
+#### Disable PortChannel
+`shutdown` 
+```
+sonic(conf-if-po1)# shutdown
+```
+
+#### Configures an IPv4 address 
+`ip address <ip-address with mask> | no ip address <ip-address>`
+```
+sonic(conf-if-po1)# ip address 2.2.2.2/24
+```
+#### Remove IPv4 address 
+`no ip address <ip-address>`
+```
+sonic(conf-if-po1)# no ip address 2.2.2.2
+```
+#### Configures an IPv6 address of the interface.
+`ipv6 address <ipv6-address with mask> | no ipv6 address <ipv6-address>`
+```
+sonic(config)# interface Ethernet 4
+sonic(conf-if-Ethernet4)# ipv6 address a::e/64
+sonic(conf-if-Ethernet4)# no ipv6 address a::e
+```
+#### Add port member
+`channel-group <channel-number>`
+```
+sonic(config)# interface Ethernet4
+sonic(conf-if-Ethernet4)# channel-group 1
+```
+#### Remove a port member
+`no channel-group`
+```
+sonic(config)# interface Ethernet4
+sonic(conf-if-Ethernet4)# no channel-group
+```
+#### Delete a PortChannel
+`no interface PortChannel <channel-number>`
+```
+sonic(config)# no interface PortChannel 1
+```
+
+#### 3.6.2.2 Show Commands
+
+##### Display summary information about PortChannels
+`show PortChannel summary`
+```
+sonic# show PortChannel summary
+
+Flags:  D - Down
+        U - Up
+--------------------------------------------------------------------------------
+Group Port-Channel           Type     Protocol  Member Ports
+--------------------------------------------------------------------------------
+1    PortChannel1    (D)     Eth      DYNAMIC    Ethernet56(D), Ethernet60(D)
+10   PortChannel10   (D)     Eth      DYNAMIC
+111  PortChannel111  (D)     Eth      DYNAMIC
+
+```
+### Show Interface status and configuration
```

Comment:
Need to display if 'fallback' is enabled/disabled.

Reply:
done